### PR TITLE
SWC-6359: add support for the new MEDIUMTEXT column type, and remove …

### PR DESCRIPTION
**Pull request by jay-hodgson**
…some dead code now that we rely on SRC to render tables/views

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Need to support cell renderers for default entity view columns, shown here:
Screen Shot 2023-01-11 at 2 58 52 PM

File: src/main/java/org/sagebionetworks/web/client/widget/table/v2/schema/ColumnModelTableRowEditorWidgetImpl.java [hunk 1]
```diff
@@ -260,12 +269,25 @@ private String getMaxSizeForType(ColumnTypeViewEnum type) {
...
   ) {
     view.setSelectionPresenter(selectionPresenter);
+
+    // add column types to the dropdown:
+    Arrays
+      .asList(ColumnTypeViewEnum.values())
+      .stream()
+      .filter((val -> (!isView || !unsupportedTypesForViews.contains(val)))) // keep all values for tables, or keep value if not found in unsupportedTypesForViews.
+      .forEach(val -> {
+        view.addColumnType(val);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/table/v2/schema/ColumnModelTableRowEditorWidgetImpl.java:289]: https://sagebionetworks.jira.com/browse/SWC-6333

Comment (jay-hodgson) [reply src/main/java/org/sagebionetworks/web/client/widget/table/v2/schema/ColumnModelTableRowEditorWidgetImpl.java:289]: Removed the list from the ui.xml, and add dynamically.  If View, do not include MediumText or LargeText

File: src/test/java/org/sagebionetworks/web/unitclient/widget/table/v2/results/facets/FacetsWidgetTest.java [hunk 2]
```diff
@@ -177,7 +176,7 @@ public void testConfigureFromQuery() {
     verify(mockView).add(mockJobTrackingWidget);
     verify(mockJobTrackingWidget)
       .startAndTrackJob(
-        eq(TableQueryResultWidget.RUNNING_QUERY_MESSAGE),
+        eq(""),
```
Comment (nickgros) [inline src/test/java/org/sagebionetworks/web/unitclient/widget/table/v2/results/facets/FacetsWidgetTest.java]: I think FacetsWidget can also be deleted

**Review by nickgros — APPROVED** [review]
Core change looks good, there might be more dead code to clean up

**Review by nickgros — APPROVED** [review]
That was a lot more dead code than I was expecting...awesome!

**Pull request merged**